src/Symfony/Component/Form/ChoiceList/Factory/CachingFactoryDecorator.php

```diff
@@ -81,7 +81,7 @@ public function getDecoratedFactory()
     /**
      * {@inheritdoc}
      */
-    public function createListFromChoices($choices, $value = null)
+    public function createListFromChoices(iterable $choices, $value = null)
```

Comment:
`callable $value = null`?

Reply:
we cannot do this, because then `string|PropertyPathInterface $value` will not be acceptable in this caching layer

Reply:
see https://github.com/symfony/symfony/pull/32237#discussion_r310611265